pages/participate/participate.md

```diff
@@ -42,7 +42,7 @@ Accepted abstracts will be placed on the website ahead of the conference.
 ## Important Dates
 
 - **Submissions open**: Monday, February 12 for all but notebooks. Notebooks to open Monday, February 26.
-- **BoFs, Workshops, and Tutorials**: Due Friday, March 22. Notifications Friday, April 26.
+- **BoFs, Workshops, and Tutorials**: Due Monday, April 1. Notifications Friday, April 26.
```

Comment:
Shoot, please also push the notifications date to May 3. I forgot these were on the pages - my bad!